buildspec.yml

```diff
@@ -15,7 +17,7 @@ phases:
       - echo Building the Docker image...
       - chmod +x ./update_data.sh
       - echo Getting latest GeoLite2 ASN DB
-      - wget https://geolite.maxmind.com/download/geoip/database/GeoLite2-ASN.tar.gz
+      - curl -O "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-ASN&license_key=${SUA_GEOLITE_ACCESS_KEY}&suffix=tar.gz"
```

Comment:
Great! Just one more necessary change and couple optional ones that I'll leave up to you.

### Necessary

* Let's specify the desired output filename explicitly using the `-o`/`--output` option instead of `-O` (upper case). I just tested this command locally again, and it successfully downloaded the file, but the saved filename was the full last portion of the URL (`geoip_download?edition_id...`). To fix that we need to use something more like:

```bash
curl -o GeoLite2-ASN.tar.gz "https://download.maxmind.com..."
```

### Optional changes

I leave the decision on these next ones to your discretion. Some other flags we could consider using are:

* `-L`/`--location` - Follow any HTTP redirects (3xx status). **Pro:** More robust to changes on their server. **Con:** On the other hand, maybe we *want* the download to fail if the URL is changed, so that we can see that something broke and we can go update it.
* If we find that the build logs look weird after this change, we could consider adding both `-s`/`--silent` and `-S`/`--show-error` so that the download progress bar isn't printed, but any errors still are. As with all short options, they can be combined, like `-sS`.